sendsmaily_subscribe/sendsmaily_subscribe.module

```diff
@@ -97,37 +96,21 @@ function sendsmaily_subscribe_configuration_page() {
     '#required' => TRUE,
   );
 
-  return system_settings_form($form);
-}
+  $form['sendsmaily_success_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Success URL'),
+    '#default_value' => variable_get('sendsmaily_success_url', ''),
+    '#description' => t('For example: http://example.com Leave blank for redirect back'),
+  );
 
-/**
- * Implements hook_form_alter().
- */
-function sendsmaily_subscribe_form_alter(&$form, &$form_state, $form_id) {
-  if ($form_id == 'user_register_form') {
-    // Add the fieldset.
-    $form['sendsmaily'] = array(
-      '#type' => 'fieldset',
-      '#title' => t('Subscribe to newsletter'),
-      '#weight' => 10,
-    );
-    // Add the checkbox to the fieldset.
-    $form['sendsmaily']['sendsmaily'] = array(
-      '#type' => 'checkbox',
-      '#title' => t('Subscribe to our newsletter'),
-    );
-    // Add submit function.
-    $form['#submit'][] = 'sendsmaily_subscribe_json_submit';
-  }
-}
+  $form['sendsmaily_failure_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Failure URL'),
+    '#default_value' => variable_get('sendsmaily_failure_url', ''),
+    '#description' => t('For example: http://example.com Leave blank for redirect back'),
```

Comment:
I would use similar text as with success URL, but with the example URL changed to something else. Maybe just https://example.com?